ui/values/localizable/fr.go

```diff
@@ -50,7 +50,8 @@ const FR = `
 "blockHeaderFetchedCount" = "%d de %d";
 "timeLeft" = "%v restant";
 "currencyConversion" = "Conversion de devise";
-"renameWalletSheetTitle" = "Renommer le porte-monnaie";
+"renameWalletSheetTitle" = "Change wallet name";
+"securityToolsInfo" = "%v Various tools that help in different aspects of crypto currency security will be located here. %v"
```

Comment:
This should be translated to French

Reply:
it doesn't necessarily have to, we would have a translator do all non-english translations

Reply:
The idea is to have a place holder for the other languages pending when we do a general sting localization like the https://github.com/planetdecred/godcr/pull/906